libr/anal/fcn.c

```diff
@@ -532,26 +536,11 @@ static int fcn_recurse(RAnal *anal, RAnalFunction *fcn, ut64 addr, ut8 *buf, ut6
 			break;
 		// TODO: use fcn->stack to know our stackframe
 		case R_ANAL_STACK_SET:
-			if ((int)op.ptr > 0) {
```

Comment:
or disabled depending on the anal plugin? (adding a new field in the RAnalPlugin struct)